plan/common_plans.go

```diff
@@ -56,6 +56,8 @@ type CheckTable struct {
 	baseSchemaProducer
 
 	Tables []*ast.TableName
+
+	GenExprs map[string]expression.Expression
```

Comment:
What about `map[int]expression.Expression` where the map key is `ColumnInfo.ID`.